src/docbuilder/rustwide_builder.rs

```diff
@@ -760,12 +760,75 @@ mod tests {
     use crate::test::wrapper;
 
     #[test]
+    #[ignore]
     fn test_build_crate() {
         wrapper(|env| {
+            let crate_ = "log";
+            let version = "0.4.14";
+            let default_target = "x86_64-unknown-linux-gnu";
+            // env.fake_release()
+            //     .name(crate_)
+            //     .version(version)
+            //     .default_target(default_target)
+            //     .add_target("i686-pc-windows-msvc")
+            //     .add_target("i686-unknown-linux-gnu")
+            //     .add_target("x86_64-apple-darwin")
+            //     .add_target("x86_64-pc-windows-msvc")
+            //     .create()?;
```

Comment:
These aren't necessary because we're doing the real thing, not going through FakeRelease, right?

Reply:
these were remains of experiments :)